UsefulStuff.Core/Features/HideFKLinesWithAxis.cs

```diff
@@ -0,0 +1,39 @@
+﻿using HarmonyLib;
+using Studio;
+using System.Reflection;
+using ToolBox;
+
+namespace HSUS.Features
+{
+    public class HideFKLinesWithAxis : IFeature
```

Comment:
There's no point in this being a feature rather than just a normal patch, right?